src/cli/commands/force/lightning/local/device/list.ts

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import { flags, FlagsConfig, SfdxCommand } from '@salesforce/command';
+import { Logger, Messages, SfdxError } from '@salesforce/core';
+import chalk from 'chalk';
+import cli from 'cli-ux';
+import { AndroidVirtualDevice } from '../../../../../../common/AndroidTypes';
+import { AndroidSDKUtils } from '../../../../../../common/AndroidUtils';
+import { CommandLineUtils } from '../../../../../../common/Common';
+import { IOSSimulatorDevice } from '../../../../../../common/IOSTypes';
+import { XcodeUtils } from '../../../../../../common/IOSUtils';
+import Setup from '../setup';
+
+// Initialize Messages with the current plugin directory
+Messages.importMessagesDirectory(__dirname);
+
+// Load the specific messages for this file. Messages from @salesforce/command, @salesforce/core,
+// or any library that is using the messages framework can also be loaded this way.
+const messages = Messages.loadMessages('@salesforce/lwc-dev-mobile', 'device');
+
+export default class List extends Setup {
+    public static description = messages.getMessage('commandDescription');
+
+    public static readonly flagsConfig: FlagsConfig = {
+        platform: flags.string({
+            char: 'p',
+            description: messages.getMessage('platformFlagDescription'),
+            longDescription: messages.getMessage('platformFlagDescription'),
+            required: true
+        })
+    };
+
+    public examples = [
+        `sfdx force:lightning:local:device:list -p iOS`,
+        `sfdx force:lightning:local:device:list -p Android`
+    ];
+
+    public async run(): Promise<any> {
+        const platform = this.flags.platform;
+        this.logger.info(`Device List command invoked for ${platform}`);
+
+        return new Promise<any>((resolve, reject) => {
+            super
+                .run() // run setup first
```

Comment:
I think this is okay for now. It'll be another area I'd like to change (probably in a minor way) when we refactor the setup code to do its main logic outside of the `run()` command—which is otherwise tightly coupled to the CLI's run loop.  Cc: @trooper2013